aldersmaler/src/main/kotlin/no/nav/pensjon/brev/alder/maler/vedlegg/alltidValgbare/Uttaksskjema.kt

```diff
@@ -0,0 +1,180 @@
+package no.nav.pensjon.brev.alder.maler.vedlegg.alltidValgbare
+
+import no.nav.pensjon.brev.alder.maler.felles.Constants
+import no.nav.pensjon.brev.api.model.maler.EmptyVedleggData
+import no.nav.pensjon.brev.template.Element.OutlineContent.ParagraphContent.Form.Text.Size
+import no.nav.pensjon.brev.template.LangBokmalEnglish
+import no.nav.pensjon.brev.template.createAttachment
+import no.nav.pensjon.brev.template.dsl.choice
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.text
+
+@TemplateModelHelpers
+val uttaksskjema = createAttachment<LangBokmalEnglish, EmptyVedleggData>(
+    title = {
+        text(
+            bokmal { +"Hvordan vil du ta ut alderspensjonen din?" },
+            english { +"How do you want to withdraw your retirement pension?" }
+        )
+    },
+    includeSakspart = false,
+    outline = {
```

Comment:
Pirk: For å ligne på syntaksen vi ellers bruker så kan outline-kroppen være utenfor funksjonskall-paranteser.